### ISSUE TITLE
Application crashes after changing piece set

### ISSUE THREAD

**Issue by Sgridev** [franciscoBSalgueiro/en-croissant#96]
Steps to reproduce:

1. Open the application
2. Change piece set

Tested on widescreen monitor - not sure it's relevant

![image](https://github.com/franciscoBSalgueiro/en-croissant/assets/35015674/7e609ae8-ee0b-4c4b-b8cc-d8840b2476ee)

**Comment by franciscoBSalgueiro** [comment]
Could you give me more details about your system? I'm not able to reproduce it on windows or linux.

**Comment by Sgridev** [comment]
Windows 11, 3840 x 1080 resolution, could it be a compatibility issue with PowerToys?

**Comment by franciscoBSalgueiro** [comment]
I also have PowerToys running so probably not. 

Could you send the logs? They're on %AppData%\org.encroissant.app\logs

**Comment by Sgridev** [comment]
Deleted the log file, changed the piece set, the application crashes, but the log file is empty :/
![image](https://github.com/franciscoBSalgueiro/en-croissant/assets/35015674/1a4f5e3e-9565-4eff-96bb-b41da2bac5dd)

Additional note:
The application maximize in a strange way before crashing.
![image](https://github.com/franciscoBSalgueiro/en-croissant/assets/35015674/380c961a-d27a-4017-8558-384c37c0d3e3)

Also it crashes only if it's not maximized, if it's maximized you can change piece set without problems

**Comment by franciscoBSalgueiro** [comment]
Does it only crash when changing the piece set? I don't really know what to make of this one. I don't see how changing the piece set is related to the screen size.

**Comment by Sgridev** [comment]
You're right, not only it crashes with piece set change, but with all the appearance changes: theme, piece set and accent color.
No problems with any of the other settings.
